Engines/deployment/splunk.py

```diff
@@ -177,7 +177,7 @@ def config_mdr(self, mdr):
         # Add correlation search setup
         if self.CORRELATION_SEARCHES:
             config["action.correlationsearch.enabled"] = "true"
-            config["action.correlationsearch.label"] = name
+            config["action.correlationsearch.label"] = name + ' - Rule'
```

Comment:
@remg427 As name has already been appended with - Rule on L41, this would make it name + ' - Rule' + '- Rule' ; could you remove the addition here ?

Reply:
thank you @behemothsecurity it was not clear that name already contained the string

```suggestion
            config["action.correlationsearch.label"] = name
```